subjects/rpn/README.md

```diff
@@ -2,15 +2,16 @@
 
 ### Instructions
 
-Write a **program** which takes a `string` that contains an equation written in `Reverse Polish Notation` (RPN) as an argument. The **program** must evaluate the equation, and then:
+Write a **program** which takes a `string` containing an equation written in `Reverse Polish Notation` (RPN). It should evaluate the equation, and print the result on the standard output followed by a newline (`'\n'`).
 
-- If the `string` is not valid or if there is not exactly one argument, `Error` must be printed on the standard output followed by a newline.
+- If the `string` is not valid, or if there is not exactly one argument, `Error` must be printed on the standard output followed by a newline.
 - If the `string` has extra spaces it is still considered valid.
 
 `Reverse Polish Notation` is a mathematical notation in which every operator follows all of its operands. In RPN, every operator encountered evaluates the previous 2 operands, and the result of this operation then becomes the first of the two operands for the subsequent operator. Operands and operators must be spaced by at least one space.
 
 The following operators must be implemented : `+`, `-`, `*`, `/`, and `%`.
 
+
```

Comment:
Not sure why you add a line here? Is something missing?

Reply:
Removed it, cheers 👍🏾.